model/oauth.go

```diff
@@ -18,17 +18,19 @@ const (
 )
 
 type OAuthApp struct {
-	Id           string      `json:"id"`
-	CreatorId    string      `json:"creator_id"`
-	CreateAt     int64       `json:"create_at"`
-	UpdateAt     int64       `json:"update_at"`
-	ClientSecret string      `json:"client_secret"`
-	Name         string      `json:"name"`
-	Description  string      `json:"description"`
-	IconURL      string      `json:"icon_url"`
-	CallbackUrls StringArray `json:"callback_urls"`
-	Homepage     string      `json:"homepage"`
-	IsTrusted    bool        `json:"is_trusted"`
+	Id                 string      `json:"id"`
+	CreatorId          string      `json:"creator_id"`
+	CreateAt           int64       `json:"create_at"`
+	UpdateAt           int64       `json:"update_at"`
+	ClientSecret       string      `json:"client_secret"`
+	Name               string      `json:"name"`
+	Description        string      `json:"description"`
+	IconURL            string      `json:"icon_url"`
+	CallbackUrls       StringArray `json:"callback_urls"`
+	Homepage           string      `json:"homepage"`
+	IsTrusted          bool        `json:"is_trusted"`
+	Scopes             Scope       `json:"scopes"`
```

Comment:
We should get the terminology consistent. If we are referring to the slice of strings as a singular scope, then the field name should also be `Scope`. But it seems like each permission attribute is already called a scope. So then we should just use `Scopes []Scope`.